examples/radiotoolbutton.py

```diff
@@ -0,0 +1,40 @@
+from gi.repository import Gtk
+
+from sugar3.graphics.radiotoolbutton import RadioToolButton
+from sugar3.graphics.radiopalette import RadioPalette, RadioMenuButton
+from sugar3.graphics.xocolor import XoColor
+import common
+
+test = common.Test()
+test.show()
+
+box = Gtk.HBox()
+test.pack_start(box, True, True, 0)
+box.show()
+
+def echo(button, label):
+    if not button.props.active:
+        return
+    print label
+
+palette = RadioPalette()
+
+button1 = RadioToolButton(icon_name='document-save', accelerator="<ctrl>S", xo_color=XoColor("#FF0000,#00FF00"), tooltip="save")
+button1.connect('clicked', lambda button: echo(button, 'document-save'))
+box.pack_start(button1, False, False, 1)
+button1.show()
+
+button2 = RadioToolButton(icon_name='document-open', accelerator="<ctrl>O", xo_color=XoColor("#0000FF,#FF0000"), tooltip="open")
+button2.connect('clicked', lambda button: echo(button, 'document-open'))
+box.pack_start(button2, False, False, 1)
+button2.show()
+
+button3 = RadioToolButton(icon_name='document-send', accelerator="<ctrl>F", xo_color=XoColor("#0000FF,#00FF00"))
+button3.connect('clicked', lambda button: echo(button, 'document-send'))
+palette.append(button3, 'menu.document-send')
```

Comment:
What is your code doing here?  I'm confused!  Please use comments to explain things that aren't the most obvious